CampusTour/Data/Data.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Data.swift
+//  CampusTour
+//
+//  Created by Ji Hwan Seung on 3/6/18.
+//  Copyright © 2018 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+struct LocationData: Codable {
+    var name: String
+    var longitude: String
```

Comment:
I thought longitude was spelled longtitude, and I spent 2 hours to figure out why the json wasn't getting parsed properly. rip

Reply:
:(

Reply:
I think it would be better if longitude and latitude were Floats or Doubles, but we would have to change how the data cleanup handles this

Reply:
I'll change the data cleanup file to make it floats